python/src/opendp/extras/examples/__init__.py

```diff
@@ -0,0 +1,76 @@
+'''
+For convenience, all the functions of this module are also available from :py:mod:`opendp.prelude`.
+We suggest importing under the conventional name ``dp``:
+
+.. code:: python
+
+    >>> import opendp.prelude as dp
+
+The methods of this module will then be accessible at ``dp.examples``.    
+'''
+
+from pathlib import Path
+
+
+def _http_get(url: str) -> bytes:
+    '''
+    Normally would use the requests library for this, but we want to avoid extra dependencies.
+
+    This only does what we need. Does not handle:
+    - plain http
+    - redirects
+    - url queries
+    - explicit port numbers
+    '''
+    from urllib.parse import urlparse
+    import http.client
+    parsed = urlparse(url)
+    assert parsed.hostname is not None
```

Comment:
Raise instead of assert is better?

Reply:
In general yes, but in this case:
- It's shorter
- I only added it to make mypy happy: It complains about the possibility of `HTTPSConnection(None)`.
- The input is something we control.
- It can make sense to use `assert` when it really is an assertion that just should be true, and not something that depends on outside input.

(But if you'd like to say that we just never use `assert`s because they can be confusing, I'd be on board with that.)